ui/utility.py

```diff
@@ -21,6 +21,14 @@ def get_page_num_width(state):
     return len(str(max_pages))
 
 
+def get_user_locale(state):
+    user = state['user']
+    if 'current_langauge' in user:
+        user['current_language']
+    else:
+        'en_GB:en'
+
+
```

Comment:
I don't think we need this function as it's just: `state['user'].get('current_language', 'en_GB:en')`?

Reply:
Ahh, I was unaware of that.  Thank you